homework_8/task_1.py

```diff
@@ -0,0 +1,5 @@
+string = input("Enter a text: ")
+print()
+for i in range(len(string)):
```

Comment:
პირდაპირ მხოლოდ ლუწ ინდექსებს გადაყოლოდი შეგეძლო for i in range(0, len(string), 2)-ით